prod-aos-rcar.yaml

```diff
@@ -380,20 +380,6 @@ parameters:
           AOS_BOARD_MODEL: "h3ulcb-4x2g-kf"
           AOS_BOARD_VERSION: "1.0"
 
-        components:
```

Comment:
Are you sure that we do not need this layer for KF? It contains drivers, board-specific changes, and modifications to IPLs (like bl2). I'm not sure we will run on KF without this layer.
Can you confirm that this change was tested on KF?

Reply:
Tested but I don't update bl2